cmd/kubernetes/kubernetes.go

```diff
@@ -133,7 +133,7 @@ func runControllerManager(machineList []string, cl *client.Client, nodeMilliCPU,
 	go util.Forever(func() { endpoints.SyncServiceEndpoints() }, time.Second*10)
 
 	controllerManager := controller.NewReplicationManager(cl)
-	controllerManager.Run(10 * time.Second)
+	controllerManager.Run(controller.TickSecs)
```

Comment:
IMHO local clarity in the replication controller trumps this, I'd find it more confusing if different clients went around multiplying it with different constants (because I really only care about this when debugging the rc, but that might be just me). I'm going to change it to DefaultSyncPeriod with no mention of the units per Daniel's suggestion, but I'll wait for you to chime in on the multiply-here-or-there bit.